modules/logger/logger.js

```diff
@@ -17,7 +17,7 @@ class Logger {
                 ];
             }
 
-            const logger = pino({
+            this.pinoLogger = pino({
```

Comment:
If we create logger in this way, then it will be used as this.pinoLogger...

Let's use factory pattern if we expect different logger types or extend the class in order to have logger as a result of constructor